create_ibu10m_items.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+# from tqdm import tqdm
+from scripts.ibu10m.item import create_ibu10m_item
+
+SAMPLE_DIR = "/Users/joshua.chung/clms-stac/scripts/ibu10m/IBU_2018_010m_al_03035_v010"
+
+
+def matching_files(directory: str):
+    tif_files = {}
+    tfw_files = {}
+    tif_vat_dbf_files = {}
+
+    for root, _, files in os.walk(directory):
+        for file in files:
+            full_path = os.path.join(root, file)
+
+            if file.endswith(".tif.vat.dbf"):
+                base_name = file[:-12]
+                tif_vat_dbf_files[base_name] = full_path
+            else:
+                base_name, extension = os.path.splitext(file)
+
+                if extension.lower() == ".tif":
+                    tif_files[base_name] = full_path
+                elif extension.lower() == ".tfw":
+                    tfw_files[base_name] = full_path
+
+    result = []
+
+    for base_name in tif_files:
+        if base_name in tfw_files and base_name in tif_vat_dbf_files:
+            result.append([tif_files[base_name], tfw_files[base_name], tif_vat_dbf_files[base_name]])
```

Comment:
Since there're always three paths I would use a tuple.
```python
result.append((tif_files[base_name], tfw_files[base_name], tif_vat_dbf_files[base_name]))
```